Jenkinsfile

```diff
@@ -106,20 +106,20 @@ pipeline {
                                 sh "rm -f ~/.xtag/status.lock ~/.xtag/acquired"
                             }
                         }
-                        stage('Run RTOS Drivers WiFi test') {
-                            steps {
-                                withTools(params.TOOLS_VERSION) {
-                                    withVenv {
-                                        script {
-                                            withXTAG(["$RTOS_TEST_RIG_TARGET"]) { adapterIDs ->
-                                                sh "test/rtos_drivers/wifi/check_wifi.sh " + adapterIDs[0]
-                                            }
-                                            sh "pytest test/rtos_drivers/wifi"
-                                        }
-                                    }
-                                }
-                            }
-                        }
+                        //stage('Run RTOS Drivers WiFi test') {
```

Comment:
Can we add a comment to explain why these tests are disabled? I wonder if there is an issue, so we can mention it in the comment.

Reply:
Done